app/assets/stylesheets/discovery-two-pane.scss

```diff
@@ -0,0 +1,3 @@
+.btn-group {
+  margin-right: 10px;
+}
```

Comment:
missing scroll on the facts, it might be very long, and we should consider search, scroll pagination etc.